test/hotspot/jtreg/compiler/valhalla/inlinetypes/InlineTypes.java

```diff
@@ -111,11 +111,12 @@ public static TestFramework getFramework() {
         protected static final String MID = ".*)+ ===.*";
         protected static final String END = ")";
         // Generic allocation
-        protected static final String ALLOC_G  = "(.*call,static  wrapper for: _new_instance_Java" + END;
-        protected static final String ALLOCA_G = "(.*call,static  wrapper for: _new_array_Java" + END;
+        protected static final String ALLOC_G  = "(.*call,static.*wrapper for: _new_instance_Java" + END;
+        protected static final String ALLOCA_G = "(.*call,static.*wrapper for: _new_array_Java" + END;
         // Inline type allocation
-        protected static final String ALLOC  = "(.*precise klass compiler/valhalla/inlinetypes/MyValue.*\\R(.*(movl|xorl|nop|spill).*\\R)*.*_new_instance_Java" + END;
-        protected static final String ALLOCA = "(.*precise klass \\[(L|Q)compiler/valhalla/inlinetypes/MyValue.*\\R(.*(movl|xorl|nop|spill).*\\R)*.*_new_array_Java" + END;
+        protected static final String MYVALUE_KLASS = "precise klass \\[(L|Q)compiler/valhalla/inlinetypes/MyValue";
```

Comment:
Just about the variable name: Shouldn't this be `MYVALUE_ARRAY_KLASS`?

Reply:
Yes, I've changed it.